doc/use_cases.md

```diff
@@ -0,0 +1,49 @@
+# Use case scenarios for posteriordb
+
+posteriordb has different uses. Below is list of example use cases we have been thinking and what features are important in each use case
+
+## Testing implementations of inference algorithms with asymptotically decreasing bias and variance
+
+If an inference algorithm has asymptotically decreasing bias and variance, then an implementation should have decreasing error compared to true posterior summaries.
+
+True summaries can be computed analytically for some simpler models, which are not necessarily that interesting from the modelling perspective, but are useful for detecting problems as the truth is known exactly. It is easiest to compare estimated and true posterior moments and quantiles for scalar quantities. For closed form marginals we can also use QQ-plots and ECDF-plots for visual exploration of unexpected discrepancies. Uniformity of PIT tests can be used to test the whole marginal, although the interpretation of different uniformity tests is more complex than point estimate comparisons.
+
+The set of applicable models can be expanded by including models which don't have closed form marginals, but we have high confidence that we can obtain draws from the true posterior. The ideal would be to use independent draws, which is feasible for some models for which the posterior factorizes to series of closed form distributions and we can obtain independent draws from each of those. We can also use MCMC for posteriors which are known to be well behaving and that given MCMC has been shown to be reliable for those posteriors. For example, dynamic HMC, is known to to work well for many models. In case of MCMC, the chains should be thinned so that the draws are approximately independent to make the further comparisons easier. When using MC and MCMC to obtain the reference posterior, a very large number of draws should be used in this use case. Monte Carlo standard errors can be used to check the accuracy of the empirical expectations. A very large number of draws could be used to compute some set of posterior moments and quantiles, and just a large number of posterior draws could be stored to enable QQ- and ECDF-plots to analyse the properties of the discrepancies. 
```

Comment:
provide examples of what a "very large number" and a "large number" mean roughly

Reply:
for posterior moments it depends on the desired MCSE, so there is no refrence
for draws to represent reference posterior we have discussed 10000 as compromise between accuracy and memory usage

Reply:
This should be made explicit otherwise people will continue to ask about what "large" means if we don't say what we mean or say that this does not mean a specific number.

Reply:
We can make the small number 10000 explicit. The larger number should be used only by people who know how to compute MCSE and can choose the number to get small enough MCSE. We can discuss what should be the accuracy for the reference posterior moments in the strict set. I think BenB went to at least 1 million when testing new U-turn criterion.